src/main/cli/cli.c

```diff
@@ -254,19 +254,7 @@ extern uint32_t inputBuffer[DSHOT_TELEMETRY_INPUT_LEN];
 extern uint32_t setDirectionMicros;
 #endif
 
-typedef enum dumpFlags_e {
-    DUMP_MASTER = (1 << 0),
-    DUMP_PROFILE = (1 << 1),
-    DUMP_RATES = (1 << 2),
-    DUMP_ALL = (1 << 3),
-    DO_DIFF = (1 << 4),
-    SHOW_DEFAULTS = (1 << 5),
-    HIDE_UNUSED = (1 << 6),
-    HARDWARE_ONLY = (1 << 7),
-    BARE = (1 << 8),
-} dumpFlags_t;
-
-typedef bool printFn(dumpFlags_t dumpMask, bool equalsDefault, const char *format, ...);
+typedef bool printFn(uint8_t dumpMask, bool equalsDefault, const char *format, ...);
```

Comment:
Looks like this now includes a revert of #7751.

Reply:
A pull request made AFTER this and then merged before - that resulted in a pain in the arse for the rebase.

Sorted now - but clean up as per 7751 out of respect should really not occur until after pending PRs are merged - especially if the PR is actually lodged and pending a merge!

Reply:
Sorry, but #7751 was decided to be put into 4.0 to facilitate the generation of  Unified Target configurations, whereas this is slated for 4.1.